src/js/service/PiskelApiService.js

```diff
@@ -259,4 +261,11 @@
   ns.PiskelApiService.prototype.onAddNewFrameEvent = function () {
     this.sendMessage_({type: MessageType.ADD_NEW_FRAME_CLICKED});
   };
+
+  ns.PiskelApiService.prototype.restrictTools = function () {
+    Constants.TOOLS_TO_RESTRICT.forEach(function (className) {
+      var elements = document.getElementsByClassName(className);
+      elements[0].classList.add('spritelab-hide');
```

Comment:
Why only the first one?

Reply:
Whoops! That was just a short-cut for developing that I didn't remove before pushing. Should be updated now. Thanks for catching!